### ISSUE TITLE
coredump during SnapshotOperations on longevity 3h job

### ISSUE THREAD

**Issue by fgelcer** [scylladb/scylla#5788]
running scylla master:
ami - ami-00eb9c3c2bb349d25
6 db nodes
version = 666.development-0.20200208.576f80be74c

```
2020-02-09 15:39:30.000: (DatabaseLogEvent Severity.CRITICAL): type=SEGMENTATION regex=segmentation line_number=27245 node=Node longevity-10gb-3h-master-db-node-e60f138f-2 [34.244.37.107 | 10.0.36.230] (seed: False)
2020-02-09T15:39:30+00:00  longevity-10gb-3h-master-db-node-e60f138f-2 !INFO    | scylla: Segmentation fault on shard 0.

void seastar::backtrace<seastar::backtrace_buffer::append_backtrace()::{lambda(seastar::frame)#1}>(seastar::backtrace_buffer::append_backtrace()::{lambda(seastar::frame)#1}&&) at /usr/include/fmt/format.h:1158
seastar::backtrace_buffer::append_backtrace() at /usr/include/fmt/format.h:1158
 (inlined by) print_with_backtrace at ./build/release/seastar/./seastar/src/core/reactor.cc:743
seastar::print_with_backtrace(char const*) at /usr/include/fmt/format.h:1158
sigsegv_action at /usr/include/fmt/format.h:1158
 (inlined by) operator() at ./build/release/seastar/./seastar/src/core/reactor.cc:3465
 (inlined by) _FUN at ./build/release/seastar/./seastar/src/core/reactor.cc:3461
?? ??:0
?? ??:0
2020-02-09 15:39:30.000: (CoreDumpEvent Severity.CRITICAL): node=Node longevity-10gb-3h-master-db-node-e60f138f-2 [34.244.37.107 | 10.0.36.230] (seed: False)
corefile_url=

backtrace=           PID: 3990 (scylla)
           UID: 996 (scylla)
           GID: 1001 (scylla)
        Signal: 11 (SEGV)
     Timestamp: Sun 2020-02-09 15:39:30 UTC (1min 49s ago)
  Command Line: /usr/bin/scylla --blocked-reactor-notify-ms 500 --abort-on-lsa-bad-alloc 1 --abort-on-seastar-bad-alloc --abort-on-internal-error 1 --log-to-syslog 1 --log-to-stdout 0 --default-log-level info --network-stack posix --io-properties-file=/etc/scylla.d/io_properties.yaml --cpuset 1-7,9-15
    Executable: /opt/scylladb/libexec/scylla
 Control Group: /scylla.slice/scylla-server.slice/scylla-server.service
          Unit: scylla-server.service
         Slice: scylla-server.slice
       Boot ID: d844d4ad4c6c4619b1cd16f507eced33
    Machine ID: df877a200226bc47d06f26dae0736ec9
      Hostname: longevity-10gb-3h-master-db-node-e60f138f-2
      Coredump: /var/lib/systemd/coredump/core.scylla.996.d844d4ad4c6c4619b1cd16f507eced33.3990.1581262770000000
       Message: Process 3990 (scylla) of user 996 dumped core.
                
                Stack trace of thread 3990:
                #0  0x0000600002a05b80 n/a (n/a)

download_instructions=
failed to upload
```

**Comment by fgelcer** [comment]
i see that previous core didn't succeed to upload, so here there is another case, on the same nemesis with a core file:

```

2020-02-10 06:54:53.000: (CoreDumpEvent Severity.CRITICAL): node=Node gemini-basic-3h-master-db-node-1765b388-3 [54.171.90.139 | 10.0.217.123] (seed: False)
corefile_url=
https://storage.cloud.google.com/upload.scylladb.com/core.scylla.996.c5fed30a976e4bd89adf1341ce41345b.4630.1581317693000000/core.scylla.996.c5fed30a976e4bd89adf1341ce41345b.4630.1581317693000000.gz
backtrace=           PID: 4630 (scylla)
          UID: 996 (scylla)
          GID: 1001 (scylla)
       Signal: 11 (SEGV)
    Timestamp: Mon 2020-02-10 06:54:53 UTC (1min 11s ago)
 Command Line: /usr/bin/scylla --blocked-reactor-notify-ms 500 --abort-on-lsa-bad-alloc 1 --abort-on-seastar-bad-alloc --abort-on-internal-error 1 --log-to-syslog 1 --log-to-stdout 0 --default-log-level info --network-stack posix --io-properties-file=/etc/scylla.d/io_properties.yaml --cpuset 0-1
   Executable: /opt/scylladb/libexec/scylla
Control Group: /scylla.slice/scylla-server.slice/scylla-server.service
         Unit: scylla-server.service
        Slice: scylla-server.slice
      Boot ID: c5fed30a976e4bd89adf1341ce41345b
   Machine ID: df877a200226bc47d06f26dae0736ec9
     Hostname: gemini-basic-3h-master-db-node-1765b388-3
     Coredump: /var/lib/systemd/coredump/core.scylla.996.c5fed30a976e4bd89adf1341ce41345b.4630.1581317693000000
      Message: Process 4630 (scylla) of user 996 dumped core.
               
               Stack trace of thread 4630:
               #0  0x0000600000a9bf28 n/a (n/a)

download_instructions=
gsutil cp gs://upload.scylladb.com/core.scylla.996.c5fed30a976e4bd89adf1341ce41345b.4630.1581317693000000/core.scylla.996.c5fed30a976e4bd89adf1341ce41345b.4630.1581317693000000.gz .
gunzip /var/lib/systemd/coredump/core.scylla.996.c5fed30a976e4bd89adf1341ce41345b.4630.1581317693000000.gz
2020-02-10 06:54:53.000: (CoreDumpEvent Severity.CRITICAL): node=Node gemini-basic-3h-master-db-node-1765b388-3 [54.171.90.139 | 10.0.217.123] (seed: False)
corefile_url=
https://storage.cloud.google.com/upload.scylladb.com/core.scylla.996.c5fed30a976e4bd89adf1341ce41345b.4630.1581317693000000/core.scylla.996.c5fed30a976e4bd89adf1341ce41345b.4630.1581317693000000.gz
```

**Comment by fgelcer** [comment]
relates to #5789

**Comment by slivne** [comment]
expect its the same as #5789 fixed by 8581617e783869ae8b57adf2ce007903ba0c9d2b - please create a new issue if there is still an issue.

Please provide the exact operation beeing executed
